akka-stream/src/main/scala/akka/stream/scaladsl/Framing.scala

```diff
@@ -18,6 +18,29 @@ import akka.util.{ ByteIterator, ByteString, OptionVal }
 
 object Framing {
 
+  /**
+   * Creates a Flow that handles decoding a stream of unstructured byte chunks into a stream of frames where the
+   * incoming chunk stream uses a specific byte-sequence to mark frame boundaries.
+   *
+   * The decoded frames will not include the separator sequence.
+   *
+   * If there are buffered bytes (an incomplete frame) when the input stream finishes and ''allowTruncation'' is set to
+   * false then this Flow will fail the stream reporting a truncated frame.
+   *
+   * @param delimiters         The byte sequences to be treated as the end of the frame.
+   * @param allowTruncation    If `false`, then when the last frame being decoded contains no valid delimiter this Flow
+   *                           fails the stream instead of returning a truncated frame.
+   * @param maximumFrameLength The maximum length of allowed frames while decoding. If the maximum length is
+   *                           exceeded this Flow will fail the stream.
+   */
+  def delimiters(
+      delimiters: List[ByteString],
```

Comment:
This should not be forced to be `List` (specific impl) but rather be `immutable.Seq[ByteString]`